README.md

```diff
@@ -1 +1,69 @@
-# PictureViewMaster
\ No newline at end of file
+## PictureViewMaster
+Interactive image projector.
+
+
+[![Build Status](https://api.travis-ci.org/inaka/EventSource.svg)](https://travis-ci.org/inaka/EventSource)
+
+```
+This library supports Swift 2.3
+Swift 2.3 is currently in branch master.
+```
+
+### Abstract
+
+This is a library that allows UIImageViews (By subclassing them on the PictureMasterImageView class) to handle a touch that will enlarge the image and let the user play with them in the ways you enable. (Dragging, rotating, zooming)
```

Comment:
In my opinion, you're giving too much detail in the abstract. You rather have to "sell your product" here by going to the point without entering into too many implementation details.
Example: "UIImageView subclass that adds functionality to drag, rotate and zoom it."